packages/web-runtime/src/services/auth/authService.ts

```diff
@@ -74,7 +79,9 @@ export class AuthService {
         ability: this.ability,
         language: this.language
       })
+    }
 
+    if (!this.userManager.areEventHandlersRegistered && !isAnonymousContext(this.router, to)) {
```

Comment:
You could make it
```
if (!isAnonymousContext(this.router, to)) {
   if (!this.userManager.areEventHandlersRegistered) {
   …
   }
  const accessToken = await this.userManager.getAccessToken()
  …
}
```
and save the duplicated `!isAnonymousContext(this.router, to)` check

Reply:
nice 👍